src/lib/modules/charts/components/model-modal/model-modal.component.ts

```diff
@@ -93,21 +101,7 @@ export class ModelModalComponent implements OnInit {
     private getClimateModels() {
         this.climateModelService.list().subscribe(data => {
             this.climateModels = data;
-
-            // Initialize 'selected' attributes with models in project
-            if (this.models.length === 0) {
-                this.selectAllClimateModels();
-            } else if (this.dataset) {
-                // dataset may be undefined for project if in form to create new project
-                this.models.forEach(projectModel => {
-                    this.climateModels.forEach(model => {
-                        if (projectModel.name === model.name) {
-                           model.selected = projectModel.selected;
-                        }
-                    });
-                });
-            }
-
+            this.selectAllClimateModels();
```

Comment:
this is because we are removing `project` in lab and always starting with all models?

Reply:
Shoot. Not sure we can remove the `[models]` input binding then as we'll no longer be able to update the models list externally like we do in Lab, meaning this change is a feature regression.I should be able to back the changes to remove the input binding, and keep the two non-breaking change fixes. Thoughts?